src/CodeGen_PTX_Dev.cpp

```diff
@@ -449,8 +449,9 @@ vector<char> CodeGen_PTX_Dev::compile_to_src() {
     PassManagerBuilder b;
     b.OptLevel = 3;
```

Comment:
Unrelated note: we should really upgrade this code to use the 'non-legacy' PassManager (Codegen_LLVM.cpp was upgraded long ago).